RestaurantApp/src/main/java/SecondApp.java

```diff
@@ -0,0 +1,5 @@
+public class SecondApp {
```

Comment:
Note about commit messages: they must contain short summary of the scope of changes. "try1" is bad message. "Delivered two submodules with "hello World" mocks" - is better.